include/api/CDataFrameAnalysisInstrumentation.h

```diff
@@ -225,6 +228,7 @@ class API_EXPORT CDataFrameTrainBoostedTreeInstrumentation final
     bool m_AnalysisStatsInitialized{false};
     std::string m_LossType;
     TLossVec m_LossValues;
+    api_t::EDataFrameTrainBoostedTreeTask m_Task{api_t::EDataFrameTrainBoostedTreeTask::E_Train};
```

Comment:
I guess we have a choice about whether to make the enum a enum class or not. Since this is the only thing in api_t my inclination is to keep as is but then I would be consistent with qualifying:
```suggestion
    api_t::EDataFrameTrainBoostedTreeTask m_Task{api_t::E_Train};
```